### ISSUE TITLE
[Bug]: the flash warning that is sent when users try to access admin (while not logged in) persists on the next visited page

### ISSUE THREAD

**Issue by r-ferrier** [geeksforsocialchange/PlaceCal#2144]
<img width="1105" src="https://github.com/geeksforsocialchange/PlaceCal/assets/46496391/a1eccc62-804d-4f17-affa-1bc5ef03e96d" alt="Screenshot 2023-11-21 at 14 15 58">

## Steps to reproduce

1. Visit admin.placecal.org
2. Visit norwich.placecal.org

## What you expected to happen

No flash message should be shown on norwich.placecal.org

**Comment by kimadactyl** [comment]
Do you think this is worth bothering with having a ticket open for? It feels less than a `v` to me.

**Comment by katjam** [comment]
It is he kind of thing that causes people to lose confidence. Agree it's not a high priority though.

**Comment by kimadactyl** [comment]
I think "school of keep every unresolved bug open", leave it open

I think "school of limit backlog to 100 tickets", its probably not in the top 200 things i would fix

**Comment by r-ferrier** [comment]
I opened it because it's a bug and our process for dealing with bugs when we find them is to create an issue. What we then do with that issue is definitely up for debate, but I think there's a good chance that if this is happening here it could be happening somewhere else that we don't know about.

**Comment by kimadactyl** [comment]
Yeah don't get me wrong it's good to have it documented, I was mostly commenting on Katja's desire to keep the backlog size down.
